sv-SE.lproj/Menu.strings

```diff
@@ -0,0 +1,258 @@
+/* 
+  Menu.strings
+  Typora
+
+ by hi@typora.io
+*/
+
+/* Typora Menu */
+
+"About Typora" = "Om Typora";
+"Preferences" = "Inställningar";
+"Check for Updates" = "Sök efter uppdateringar";
+"Services" = "Tjänster";
+"Hide Typora" = "Göm Typora";
+"Hide Others" = "Göm andra"; /* Context? */
+"Show All" = "Visa alla"; /* Context? "Visa allt" may be a better translation depending on context */
+"Quit Typora" = "Avsluta Typora";
+
+/* File Menu */
+
+"File" = "Arkiv";
+"New" = "Ny";
+"Open" = "Öppna";
+"Open Folder" = "Öppna mapp";
+"Reopen with Encoding" = "Ladda om med kodning";
+"Auto" = "Auto";
+"Open Recent" = "Öppna tidigare";
+"Clear Items" = "Rensa";
+"No Recent Files" = "Inga tidigare filer";
+"Open Quickly" = "Öppna snabbt";
+"Open File Location" = "Öppna filsökväg";
+"Reveal in Library" = "Visa i bibliotek";
+"Reveal in File Tree" = "Visa i filträd";
+"Close" = "Stäng";
+"Save" = "Spara";
+"Save As" = "Spara som";
+"Duplicate" = "Duplicera";
+"Rename" = "Byt namn";
+"Move To" = "Flytta till";
+"Revert To" = "Återställ till";
+"Browse All Versions" = "Se alla versioner";
+"Import" = "Importera";
+"Export" = "Exportera";
+"Page Setup" = "Sidinställning";
+"Print" = "Skriv ut";
+
+/* Edit Menu */
+"Edit" = "Redigera";
+"Undo" = "Ångra";
+"Redo" = "Gör om";
+"Cut" = "Klipp ut";
+"Copy" = "Kopiera";
+"Paste" = "Klistra in";
+"Copy as Markdown" = "Kopiera som Markdown";
+"Copy as HTML Code" = "Kopiera som HTML";
+"Paste as Plain Text" = "Klistra in utan formattering";
+"Select All" = "Välj allt";
+"Select Line/Sentence" = "Välj rad/mening";
+"Select Styled Scope" = "Välj Styled Scope"; /* What is meant by a 'styled scope'? /*
```

Comment:
Example:

> This is a text **with strong style**

When user move cursor inside "with strong style", and click this menu item, all test between "**" ("with strong style") will be selected.